pkg/controller/scheduling/scheduling.go

```diff
@@ -70,6 +70,12 @@ func addScheduling(req router.Request, appInstance *v1.AppInstance, workloads ma
 		}
 
 		for sidecarName, sidecarContainer := range container.Sidecars {
+			// disable extra resource indication (GPU et al. for sidecars)
+			if computeClass != nil {
+				if computeClass.Resources != nil {
+					computeClass.Resources = &corev1.ResourceRequirements{Limits: corev1.ResourceList{}, Requests: corev1.ResourceList{}}
+				}
+			}
```

Comment:
```suggestion
			if computeClass != nil && computeClass.Resources != nil {
				computeClass.Resources = &corev1.ResourceRequirements{}
			}
```

Reply:
fixed. Don't know how you prefer me to click on resolve or you'll do it.